save-agent/src/linuxX64Main/kotlin/org/cqfn/save/agent/SaveAgent.kt

```diff
@@ -156,7 +157,14 @@ class SaveAgent(internal val config: AgentConfiguration,
 
     @Suppress("MagicNumber")
     private fun runSave(cliArgs: String): ExecutionResult = ProcessBuilder(true, FileSystem.SYSTEM)
-        .exec(config.cliCommand.let { if (cliArgs.isNotEmpty()) "$it $cliArgs" else it }, "", config.logFilePath.toPath(), 100_000L)
+        .exec(
+            config.cliCommand.let {
+                if (!it.contains(STANDARD_TEST_SUITE_DIR)) "$it $cliArgs" else it
```

Comment:
`$cliArgs` here could be not empty only in the Git mode and this variable contain the `testRootPath` + set of tests in those case

in standard mode just use command, which we created in `DockerService.kt`